src/staff_explode.lua

```diff
@@ -72,6 +75,12 @@ function ensure_score_has_enough_staves(slot, note_count)
 end
 
 function staff_explode()
+    if finenv.QueryInvokedModifierKeys and
+    (finenv.QueryInvokedModifierKeys(finale.CMDMODKEY_ALT) or finenv.QueryInvokedModifierKeys(finale.CMDMODKEY_SHIFT))
```

Comment:
Going to approve and merge these scripts, though we might consider moving this into the library if it'll be used often.

For instance, I just added the "client" library to help check what a machine is capable of (https://www.finalelua.com/docs/library/client). Something similar might be good for this like `client.is_user_pressing_modifier_keys()` or similar.

Not that it saves much typing, but if this behavior will be used elsewhere, moving it to a library function will ensure consistency amongst all scripts that use it.